code/datums/uplink/grenades.dm

```diff
@@ -34,6 +34,11 @@
 	item_cost = 10
 	path = /obj/item/weapon/storage/box/explosive
 
+/datum/uplink_item/item/grenades/incendiary
+	name = "5xIncendiary Grenades"
+	item_cost = 15
+	path = /obj/item/weapon/storage/box/incendiary
```

Comment:
Atmos caused mass murder happening too often already a problem on server, so let's not add this.

Reply:
Done